deploy/ngsadev/istio/istio-gateway.yaml

```diff
@@ -16,4 +16,5 @@ spec:
       serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
       privateKey: /etc/istio/ingressgateway-certs/tls.key
     hosts:
-    - ${ASB_DOMAIN}
+    - "ngsa/ngsa-pre-central-dev.cse.ms"
```

Comment:
should be  `ngsa/{ASB_DOMAIN}` and `monitoring/{ASB_DOMAIN}` since this is part of the template

Reply:
disregard my message :) it is correct the way it is